src/IndexGame.py

```diff
@@ -29,9 +28,46 @@
             }
 mapSize = [0,0]  #X-axis, Y-axis
 
+MainMenuData = \
+['Welcome, mayor of Simp City!',
+'------------------------------',
+'1. Start new game',
+'2. Load saved game',
+'3. Show high scores\n'
+'0. Exit']
+
 # General functions
-def displayMainMenu():
-    return True
+def displayMainMenu(): # To display the main menu
+    for i in range(len(MainMenuData)):
+            print(MainMenuData[i])
+   
+def runMainMenu(): # To run the main menu options
+    result = 1
+    while result != 0:
+        displayMainMenu()
+        userInput = input('Enter your option: ')
+        result = MainMenuSelection(userInput)
+
+def MainMenuSelection(userInput): # To validate the user main menu options
+        if userInput == "0":
+            exit = exitGame()
+            print(exit)
+            return 0
+        elif userInput == "1":
+            gameinit.newGame()
```

Comment:
The new way for calling the new game is in the original file.

Reply:
Can change the gameinit.newGame() as stated above